packages/telemetry/src/otel-trace.js

```diff
@@ -31,7 +31,7 @@ export const makeOtelTracingProvider = opts => {
   const resource = new Resource(getResourceAttributes(opts));
 
   const exporter = new OTLPTraceExporter();
-  console.info('Enabling OTLP Traces Exporter to', exporter.getDefaultUrl({}));
+  console.info('Enabling OTLP Traces Exporter');
```

Comment:
From the [opentelemetry-js 0.54.0 changelog notes](https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/CHANGELOG.md#0540):
> (user-facing) <code>getDefaultUrl</code> was intended for internal use has been removed from all exporters